server/locale/DE/GPL/mutations/gplq_eqz.js

```diff
@@ -1,8 +1,74 @@
 import Logo from '../../../../message/logos';
 import { xSmallFallback, textWrap, messageLogoWidth, altNoWrap, setLogoTop } from '../../../../message/mediaQueries';
-import { textLogoMutations } from '../../../../message/logoMutations';
+import { flexLogoMutations, textLogoMutations } from '../../../../message/logoMutations';
+
+const headlineBreaks = [
+    {
+        sizes: ['xsmall'],
+        breaks: ['Ratenzahlung']
+    },
+    {
+        sizes: ['medium'],
+        breaks: ['Jahreszins: ab']
+    }
+].reduce((acc, item) => {
+    const { sizes, breaks } = item;
+    sizes.forEach(size => {
+        acc.push({
+            tag: size,
+            br: breaks
+        });
+    });
+    return acc;
+}, []);
+
+const flex = [
+    [
+        'default',
+        {
+            logo: Logo.PP_PAYPAL.WHITE,
+            headline: [
+                {
+                    tag: 'xsmall'
+                },
+                {
+                    tag: 'medium'
+                }
+            ],
+            disclaimer: ['default']
+        }
+    ],
+    [
+        'ratio:20x1',
+        {
+            styles: [
+                `
+                .message__messaging  {
+                    padding-right: 12%;
+                }
+                `
+            ],
```

Comment:
We won't need this in the 8x1 and 20x1 sections anymore with some of the css suggestions added in. 
```suggestion
```